libs/src/services/discoveryProvider/index.js

```diff
@@ -618,8 +612,7 @@ class DiscoveryProvider {
       endpoint: 'users/account',
       queryParams: { wallet }
     }
-    // TEMP: Forcing 1 retry to see recursion logic
-    return this._makeRequest(req, 1, true)
+    return this._makeRequest(req, 0, true)
```

Comment:
0?

Reply:
i forced the retry to be 1 to see if the retry logic was correct. the retry count was originally 0. gh probably didnt recognize that the revert isn't actually a change? 
https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/services/discoveryProvider/index.js

Reply:
doesn't making this 0 here force autoselection every time?